code/modules/surgery/bodyparts/dismemberment.dm

```diff
@@ -0,0 +1,314 @@
+
+
+//Dismember a limb
+/obj/item/bodypart/proc/dismember()
```

Comment:
This really needs to spray blood/friendship/*insert internal body fluid here* given that you're cutting off a limb.